.github/ISSUE_TEMPLATE/membership.md

```diff
@@ -7,28 +7,32 @@ assignees: ''
 
 ---
 
+<!-- Please remember to change the title of this issue by replacing
+ <your-GH-handle> with the actual GitHub handle -->
+
 ### GitHub Username
 
 e.g. (at)example_user
 
-### Organization you are requesting membership in
-
-e.g. (at)open-telemetry
-
 ### Requirements
 
 - [ ] I have reviewed the community membership guidelines (https://github.com/open-telemetry/community/blob/master/community-membership.md)
 - [ ] I have enabled 2FA on my GitHub account. See https://github.com/settings/security
 - [ ] I have subscribed to the [Gitter channel](https://gitter.im/open-telemetry/community)
 - [ ] I am actively contributing to 1 or more OpenTelemetry subprojects
-- [ ] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
+- [ ] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines. Among other requirments, sponsors must be approvers or maintainers of at least one repository and not being affiliated with a single company
 - [ ] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application
 
 ### Sponsors
 
+<!-- Replace (at) with the `@` sign -->
+
 - (at)sponsor-1
 - (at)sponsor-2
 
+**New!**: Each sponsor should reply to this issue with the comment "*I support*".
```

Comment:
Do we need to mention "New"? People apply once for membership (ignoring the rare case of rejection), so they probably don't care what is new vs old.

Reply:
this is new for sponsors. Some sponsors do not know what to do, some was doing it differently for a while now. So I wanted to emphasize it. Are you and @arminru strongly against?

Reply:
I'm not sure if "repeat sponsors" will notice it more likely with the "New!" but I'm not strongly opposed, feel free to keep it.

Reply:
people had been using reactji to support :) which I feel also is fine

Reply:
reactji do not produce a notification in inbox. Harder to track whether there were any updates on the issue